### PR TITLE
Ffs/feed

### PR DESCRIPTION
**Pull request by Brennaleker**
:shipit:  WIP - styling

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ gem "omniauth"
 gem "omniauth-instagram"
 gem "omniauth-vimeo"
 gem "bootstrap-sass"
+gem 'google-webfonts'
```
Comment (sojeri) [inline Gemfile:9]: sweet!

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -62,3 +62,98 @@
...
+  color: #080;
+}
+
+a:hover{
+  color: #66b766;
+}
+
+html, body {
+  background: url('congruent.png');
+}
```
Comment (sojeri) [inline app/assets/stylesheets/application.css.scss:158]: so many sweet styles!

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -10,7 +10,7 @@
   <body>
     <header id="top" class="nav navbar-default">
       <div class="col-sm-2">
-        <h2><%= link_to "FriendFace", root_path %></h2>
+        <h2 class="brand"><%= link_to "Friendface", root_path %></h2>
```
Comment (sojeri) [inline app/views/layouts/application.html.erb:13]: yay!

**Comment by sojeri** [comment]
I think these changes look amazing, but the best part is clearly your staff photo.

File: app/assets/stylesheets/application.css.scss (hunk 2 above)
Comment (vikshab) [inline app/assets/stylesheets/application.css.scss:158]: Good job!

**Pull request merged**